framework/dependencies.yml

```diff
@@ -69,7 +69,7 @@ require:    &allDependencies
     - org.apache.logging.log4j -> log4j-slf4j2-impl 2.23.0
     - org.yaml -> snakeyaml 2.2
     - net.spy -> spymemcached 2.12.3
-    - com.thoughtworks.xstream -> xstream 1.4.20
+    - com.thoughtworks.xstream -> xstream 1.4.21
```

Comment:
this is an actual upgrade, to avoid [CVE-2024-47072](https://cve.mitre.org/cgi-bin/cvename.cgi?name=CVE-2024-47072)